src/hooks/useAppBootstrap.js

```diff
@@ -72,9 +91,7 @@ export const useAppBootstrap = client => {
   useEffect(() => {
     if (initialRoute !== 'fetching' && isLoading) {
       setIsLoading(false)
-      if (initialRoute.route !== routes.home) {
```

Comment:
Why did you need to remove this?

Reply:
I'm wondering about the side effects, so please double check you didn't break the splashscreen management that should happens only after app and konnector tiles are loaded on the home.

Reply:
I agree it makes little sense to remove it. I do not remember why exactly I removed this. Sometime splash screen did not hide. But I tried again and it works with it so I add it back.